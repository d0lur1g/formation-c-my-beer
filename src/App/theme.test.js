import theme from "./theme";

describe("theme", () => {
  it("returns a correct settings of values", () => {
    expect(theme).toMatchSnapshot();
  });
});
