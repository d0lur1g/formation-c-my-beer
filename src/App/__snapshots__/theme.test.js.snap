// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theme returns a correct settings of values 1`] = `[Function]`;
